client/state/action-log/README.md

```diff
@@ -0,0 +1,38 @@
+# Action Logger
+**A Redux [store enhancer](https://github.com/reactjs/redux/blob/master/docs/Glossary.md#store-enhancer)**
+
+The Action Logger is a Redux store enhancer which records the most recently dispatched redux action
+so they can be inspected in the development environment for debugging and interactive design.
+
+## Functionality
+
+### Action logging
+
+Like the [Redux DevTools](https://github.com/zalmoxisus/redux-devtools-extension), the console dispatcher can also track a history of the actions flowing through the store.
```

Comment:
I thought we're separating the concepts of `console dispatcher` and `action logger` here - if that's the case, should we replace `console dispatcher` with `action logger`?